lib/src/__tests__/DatePicker.test.tsx

```diff
@@ -24,7 +22,7 @@ describe('e2e - DatePicker default year format', () => {
       <DatePicker
         renderInput={props => <TextField {...props} />}
         DialogProps={{}}
-        PopoverProps={{}}
+        PopperProps={{}}
```

Comment:
It seems we don't need these two props.

Reply:
For now, it is only checked that this prop is available. Should be moved to typescript.

Reply:
Agree, let's move it to a different place. We should keep the tests as focused as possible. Testing one thing at the time.